internal/eventstream/api_responses.go

```diff
@@ -0,0 +1,261 @@
+package eventstream
```

Comment:
I'd rename this package (and directory). For this project, it's more relevant that we're interacting with Icinga 2 to make it work as a source, not that this is done using its event stream API. Maybe just go for `icinga2`, alternative I have in mind would be 'icinga2source` (looks cumbersome) or maybe something like `internal/sources/icinga2/` (looks like we'd want to implement more in this style, which I don't think we want).